config/runtime.exs

```diff
@@ -91,10 +91,29 @@ default_locale =
 config :gettext, :default_locale, default_locale
 config :timex, :default_locale, default_locale
 
-# enable brainstorming teasers:
+delete_brainstormings_after_days =
+  if System.get_env("MW_FEATURE_BRAINSTORMING_REMOVAL_AFTER_DAYS") &&
```

Comment:
Not a big issue, but `export MW_FEATURE_BRAINSTORMING_REMOVAL_AFTER_DAYS="        "` will break.

`export MW_FEATURE_BRAINSTORMING_REMOVAL_AFTER_DAYS="gerardo"` will also break. But this is ok as people should be forced to use valid settings. 

`export MW_FEATURE_BRAINSTORMING_REMOVAL_AFTER_DAYS="10        "` will also break because of the trailing whitespaces.

Reply:
i can add a trim